.github/scripts/check_skip_ci.sh

```diff
@@ -16,35 +27,40 @@ set -euo pipefail
 files_to_check=$(git diff --name-only "$(git merge-base origin/$SKIP_CHECK_BRANCH HEAD~)"...HEAD)
 
 # Define the directories to check
-skipped_directories=("assets" ".changelog/")
+skipped_directories=("assets" ".changelog/", "version", ".github")
```

Comment:
```suggestion
skipped_directories=("assets" ".changelog/", "version")
```
if we change workflows, this means that CI would not run to assert them

Reply:
addressed